corehq/apps/sms/views.py

```diff
@@ -2151,12 +2151,12 @@ def page_context(self):
             )
         else:
             wa_active_backend = turn_backend.get() if turn_backend.count() else infobip_backend.get()
-            try:
-                templates = wa_active_backend.get_all_templates()
+            templates = wa_active_backend.get_all_templates()
+            if templates:
                 for template in templates:
                     template['template_string'] = wa_active_backend.generate_template_string(template)
                 context.update({'wa_templates': templates})
-            except TypeError:
+            else:
```

Comment:
Possible subtle change in behavior here: previously it appears to have been acceptable for the list of templates to be empty (and that would not cause an error). Now it will display an error if `templates` is an empty sequence (list or tuple). Does that matter?

If the above code suggestion (`templates is not None`) is acceptable then this question can be ignored.